src/api/getLanglinks.js

```diff
@@ -1,9 +1,14 @@
 import {
   buildMwApiUrl, cachedFetch,
   isSupportedForReading, getDirection,
-  prioritizedList
+  isPrioritized
 } from 'utils'
 
+const convertToWikiLang = lang => {
+  // The lang link endpoint return 'nb' but the wiki subdomain is 'no'
```

Comment:
This is not future proof as there is likely going to be more exceptions in the future.

Why are you not using the `url` prop, e.g. https://en.wikipedia.org/wiki/Special:ApiSandbox#action=query&format=json&prop=langlinks&titles=London&llprop=url%7Clangname%7Cautonym&lllimit=500

Reply:
Good idea. I did it in the latest commit.